libindy/src/services/metrics/mod.rs

```diff
@@ -125,10 +150,8 @@ mod test {
 
         metrics_service.append_command_metrics(&mut metrics_map);
 
-        assert_eq!(metrics_map.len(), COMMANDS_COUNT * 4);
-        assert!(metrics_map.contains_key("issuer_command_create_schema_queued_commands_count"));
-        assert!(metrics_map.contains_key("issuer_command_create_schema_queued_commands_duration_ms"));
-        assert!(metrics_map.contains_key("issuer_command_create_schema_executed_commands_count"));
-        assert!(metrics_map.contains_key("issuer_command_create_schema_executed_commands_duration_ms"));
+        assert!(metrics_map.contains_key("commands_count"));
+        assert!(metrics_map.contains_key("commands_duration"));
+        assert_eq!(metrics_map.get("commands_count").unwrap().as_array().unwrap().len() + metrics_map.get("commands_duration").unwrap().as_array().unwrap().len(), COMMANDS_COUNT * 4);
```

Comment:
It would be much better to test number of `command_count ` and `commands_duration_ms` metrics separately (checking that each are `COMMANDS_COUNT * 2`)

Also, this test is definitely not complete now - we should test internal schema as well (i.e. that metrics include array of objects, and each object has value and tags, and for timing metrics tags are objects with fields like command, subcommand and stage, and stage is either queued or executed, etc etc)